app/controllers/bikes_controller.rb

```diff
@@ -28,6 +28,9 @@ def index
 
   def show
     @components = @bike.components
+    if @bike.deleted?
```

Comment:
This should actually go in the `find_bike` method which already has something similar